search-results.hbs

```diff
@@ -10,6 +10,7 @@
 </main>
 
 <script>
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
It's not the DRYest, but this seems to fix the issue:

```suggestion
document.addEventListener('DOMContentLoaded', () => {
    const client = algoliasearch('QMJYL5WYTI', '89770b24481654192d7a5c402c6ad9a0');
    const index = client.initIndex(main['algolia-index-name']);
```

There may be a way to set this up globally, but it may be tricky with the deferred Algolia scripts and the `DOMContentLoaded` events.

Reply:
This seems good.  I do get a few errors and a 404 on the /search page, for both this and `main`.  Is there a trick to getting that working locally?